Mark Xamarin as end of support (#3725)
OWASP/owasp-mstg: techniques/android/MASTG-TECH-0011.md

> Once you have set up an interception proxy and have a MITM position, you might still not be able to see anything. This is mainly due to the following reasons:

Reviewer: ```suggestion
Even after setting up an interception proxy and achieving a MITM position, you may still not be able to see any HTTP requests. This is mainly for the following reasons:
```

> - The app is using a cross-platform framework that does not use the proxy settings of the Android OS or

Reviewer: ```suggestion
- The app uses a cross-platform framework that does not use the Android OS proxy settings.
```
